src/utils/ValidationHelper.js

```diff
@@ -14,3 +14,23 @@ export const validateTarget = (target) => {
   }
   return errorMessage;
 };
+
+export const validateUserProfile = (settings) => {
+  let errorMessage;
+  if (settings.email === 0) {
+      errorMessage = "You must enter an email.";
+  }
+  if (settings.currentPassword === 0) {
```

Comment:
you should use `else if` for the following `if`s in order to avoid checking each condition